tgui/packages/tgui-panel/chat/model.js

```diff
@@ -14,9 +14,7 @@ export const canPageAcceptType = (page, type) => (
 export const createPage = obj => {
   const acceptedTypes = {};
   for (let typeDef of MESSAGE_TYPES) {
-    if (typeDef.important) {
-      acceptedTypes[typeDef.type] = true;
-    }
+    acceptedTypes[typeDef.type] = true;
```

Comment:
huh. it's making them all false for me, can you repro? 
![image](https://github.com/goonstation/goonstation/assets/4741640/ae36011b-c886-40fd-896e-aa8ffbe55f64)

Reply:
it works fine when creating a new tab, again old ones will not update its set on creation.
![image](https://github.com/goonstation/goonstation/assets/113442598/6491b0e0-626c-4562-aa46-672b0f3357c6)